modules/cco-ccoctl-deleting-sts-resources.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/uninstalling-cluster-aws.adoc
+
+[id="cco-ccoctl-deleting-sts-resources_{context}"]
+= Deleting AWS resources with the Cloud Credential Operator utility
```

Comment:
Do you need "utility" at the end? I'm guessing so but I don't recall seeing this for other Operators, so just asking.

Reply:
yeah, the utility is a separate thing - thanks :)